benchmarking/runs/multiple_tokenization_goals.py

```diff
@@ -0,0 +1,188 @@
+import math
+
+import typer
+
+from api.models.enums import ScenarioAttribute, Gender, Gpa, Age, Race
+from benchmarking.data.simulated_data.mock_student_provider import (
+    MockStudentProviderSettings,
+    MockStudentProvider,
+)
+from benchmarking.evaluations.goals import DiversityGoal
+from benchmarking.evaluations.graphing.graph_metadata import GraphData
+from benchmarking.evaluations.graphing.line_graph import line_graph
+from benchmarking.evaluations.graphing.line_graph_metadata import LineGraphMetadata
+from benchmarking.evaluations.metrics.average_gini_index import AverageGiniIndex
+from benchmarking.evaluations.metrics.average_gini_index_multi_attribute import (
+    AverageGiniIndexMultiAttribute,
+)
+from benchmarking.evaluations.metrics.maximum_gini_index import MaximumGiniIndex
+from benchmarking.evaluations.metrics.minimum_gini_index import MinimumGiniIndex
+from benchmarking.evaluations.metrics.priority_satisfaction import PrioritySatisfaction
+from benchmarking.evaluations.scenarios.multiple_tokenization_goals import (
+    MultipleTokenizationGoals,
+)
+from benchmarking.simulation.goal_to_priority import goals_to_priorities
+from benchmarking.simulation.simulation import Simulation
+
+
+def concentrate_multiple_max_3(num_trials: int = 10):
+    """
+    Goal: Run a scenario with multiple tokenization constraints to benchmark the algorithm
+    """
+
+    # Defining our changing x-values (in the graph sense)
+    class_sizes = list(range(50, 501, 50))
+    num_trials = 10
```

Comment:
unneeded, num_trials comes from the function arguments
(pls quick check other runs in case we made this mistake elsewhere too)